tests/integration/test_properties.py

```diff
@@ -66,8 +68,8 @@ def test_archive_property_by_id(client, auth_headers, new_property, test_databas
     assert responseBadPropertyID.get_json() == {'message': 'Property cannot be archived'}
     assert responseBadPropertyID.status_code == 400
 
-def test_delete_property_by_name(client, auth_headers, new_property, test_database):
-    test_property = PropertyModel.find_by_name(new_property.name)
+def test_delete_property_by_name(client, auth_headers, test_database):
+    test_property = PropertyModel.find_by_name("The Reginald")
```

Comment:
If the name is 'test1', this test would failed because there are two properties in test database with 'test1'. This is being fix by https://github.com/codeforpdx/dwellingly-app/issues/395